src/Maui.RevenueCat.Android/Maui.RevenueCat.Android.csproj

```diff
@@ -23,7 +23,7 @@
 		<DebugSymbols>true</DebugSymbols>
 		<DebugType>embedded</DebugType>
 		<PackageIcon>icon.png</PackageIcon>
-		<Version>7.12.0.0</Version>
+		<Version>7.13.0.0</Version>
```

Comment:
version is derived from the native Android library used. V`7.12.0`.
So please change it as `7.12.0.1` where 4th number is the number of patch for this version